cmd/manager/main.go

```diff
@@ -37,8 +36,14 @@ import (
 	"github.com/operator-framework/operator-sdk/pkg/metrics"
 	sdkVersion "github.com/operator-framework/operator-sdk/version"
 	"github.com/spf13/pflag"
+	appsv1 "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
```

Comment:
I'm not sure I understand why we need Azure, GCP, OIDC...

Reply:
Hmmm, i'm not too sure, i was organizing the imports and thought i would move the comment with the associated import. However, since i'm no longer touching this file i've undone my changes there - hope that is okay.

Reply:
I missed it was already there, my fault :man_facepalming:

Reply:
:smile: no worries! sorry for the confusion!